src/main/kotlin/day05/Collection.kt

```
@@ -0,0 +1,40 @@
+) {
+
+}
+
+class Collection {
+
+}
+
+class Test {
+    fun attack() {
+
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)